akvo/rsr/static/tests/scripts-src/my-results/utils.test.js

```diff
@@ -120,6 +106,27 @@ describe("function getCookie", () => {
     });
 });
 
+describe("function _", () => {
+    const store = mockStore({
+        page: {
+            strings: {
+                actual: "Actual",
+                actual_comment: "Actual comment"
+            }
+        }
+    });
+    beforeEach(() => {
+        mock(store);
```

Comment:
Using rewire leads to mock() not having to be exported from store.js. Use rewire instead to be able to use it in the tests.